src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,54 @@
+@import "../variables";
+
+.clock {
+  position: relative;
+  border: 2px gray solid;
+  height: $size;
+  width: $size;
+  border-radius: 50%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__mid {
+    height: 5vh;
```

Comment:
Make it depends of for example by clock or clock hand to make it more addaptive to changes.
Right now width and height of mid-clock-part stay same if size of clock changed.